js/app.js

```diff
@@ -68,9 +39,6 @@ $(function(){
 
   // menu bar
 
-  var prev;
-  var n = 0;
-
   var headings = $('h2, h3').map(function(i, el){
```

Comment:
I’m not really sure if there’s a reason for it not to be CSS. I’ll try it in the next PR. This one focused on removing unused stuff to avoid noise in the other two PRs.